webapp/src/css/enketo/medic.less

```diff
@@ -18,10 +18,21 @@
     #form-languages {
       display: none;
     }
-    .option-wrapper {
+
+    .dropdown-menu .option-wrapper {
+      padding: 3px;
       .option-label {
+        width: calc(100% - 34px);
+      }
+    }
+
+    .question:not(.or-appearance-likert):not(.or-appearance-columns) .option-wrapper {
+      padding: 5px;
```

Comment:
I think I prefer it without this padding as it means we can use 10px more width which is useful on mobile. Why is this needed?

Reply:
I tried to fix something with that, but I can't remember anymore nor reproduce anything weird by not having it. 
I'm just going to remove it.